src/js/pages/settings/UISettingsPage.js

```diff
@@ -56,6 +56,16 @@ class UISettingsPage extends React.Component {
         />
         <div className="container">
           <ConfigurationMap>
+            <ConfigurationMapHeading className="flush-top">
+              <Trans>DC/OS UI Details</Trans>
+            </ConfigurationMapHeading>
+            <ConfigurationMapSection>
+              <ConfigurationRow
+                keyValue="installedVersion"
+                title={<Trans>Installed Version</Trans>}
+                value={window.DCOS_UI_VERSION}
```

Comment:
🤷‍♂️

Reply:
Temporary solution.

Reply:
That is what we've decided to go with. The issue wasn't even originally planned so scoping in the data layer would have been a massive creep.